src/main/java/Deadline.java

```diff
@@ -0,0 +1,26 @@
+public class Deadline extends Todo {
+    protected String by;
```

Comment:
`by` could come off as an ambiguous/awkward sounding field name. Perhaps something clearer like "date" or "time" would be better.